docs/manual/common/guide/devmode/DevEnvironment.md

```diff
@@ -96,11 +96,6 @@ You can verify that your services are running by viewing `http://localhost:9008/
     "url":"tcp://127.0.0.1:4000/cas_native",
     "portName":null
   },
-  {
-    "name":"kafka_native",
-    "url":"tcp://localhost:9092/kafka_native",
-    "portName":null
-  },
```

Comment:
Really? How does DevMode work if not locating `kafka_native` on the DevMode service registry ?